client/src/pages/User.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import API from "../utils/API";
+import { Col, Row, Container } from "../components/Grid";
+
+function User() {
+
+  // Loads all books and sets them to books
+  function loadUserRoutes() {
+    API.getUser()
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.getUserSettings(1)
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.updateUserSettings(1)
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.saveUser({})
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.getDashData(1)
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.getNewChallenge()
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.getMovieSugg()
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.getPhysActSugg()
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.getMentalActSugg()
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+    API.updateChall()
+      .then(res =>
+        console.log(res)
+      )
+      .catch(err => console.log(err));
+  };
+
+  loadUserRoutes()
+
+
```

Comment:
@ShepLT1 can you put comments in for each route? (what they will take in and what they return)

Reply:
actually, why do you need to load all the routes here? why not just keep them in the API until they are needed?

Reply:
just to check that they were working. I can take it out now

Reply:
You can leave them in commented out with an explanation too if you think they are good to have for testing